pysimplesoap/__init__.py

```diff
@@ -12,5 +12,8 @@
 
 TIMEOUT = 60
 
-
-from . import client, server, simplexml, transport
+# to prevent errors during installation
```

Comment:
"setup.py" imports transport.py to retrieve the name and version number of the application.
The problem is, transport.py imports "urllib3" which presumably would not have been installed by the time "transport.py" is initially imported.